pest_derive/Cargo.toml

```diff
@@ -17,7 +17,8 @@ proc-macro = true
 [dependencies]
 quote = "^0.3"
 syn = "^0.10"
-pest = { git = "https://github.com/pest-parser/pest", branch = "procedural" }
+# pest = { git = "https://github.com/pest-parser/pest", branch = "procedural" }
```

Comment:
Of course. Just make sure you keep the local paths local only without pushing it to the branch since then it would make depending on the git branch in Cargo impossible. (For those hacking on the alpha)